account/src/main/java/com/user/account/services/TransactionService.java

```diff
@@ -0,0 +1,12 @@
+package com.user.account.services;
+
+import java.util.ArrayList;
+
+import com.user.account.entity.Transactions;
+
+public interface TransactionService {
+	
+	public Transactions addTransaction(Transactions newTransaction);
+	
+	public ArrayList< Transactions> getSummary(Long accountNumber);
```

Comment:
why `ArrrayList` and not `List`?

We should use abstract class everywhere it's possible